extras/scripts/create_ingredients_from_foodfacts.py

```diff
@@ -0,0 +1,71 @@
+from pymongo import MongoClient
+import os
+import django
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings")
+django.setup()
+from wger.nutrition.models import Ingredient  # noqa: E402
+from wger.core.models import Language  # noqa: E402
+from django.utils import timezone  # noqa: E402
+
+client = MongoClient(port=27017)
+db = client.off
+
+langs = {"en": "English", "de": "German", "es": "Spanish",
+         "ru": "Russian", "fr": "French", "bg": "Bulgarian",
+         "el": "Greek", "nl": "Dutch", "no": "Norwegian", "cs": "Czech",
+         "sv": "Swedish", "pt": "Portuguese"}
+
+for product in db.products.find({'lang': {"$in": list(langs.keys())}}):
+    lang = product['lang']
+
+    main_details = ['product_name', 'code']
+    if all(req in product for req in main_details):
+        name = product['product_name']
+        code = product['code']
+
+    required = ['energy-kcal_100g', 'proteins_100g', 'carbohydrates_100g',
+                'sugars_100g', 'fat_100g', 'saturated-fat_100g']
+    if all(req in product['nutriments'] for req in required):
+        energy = product['nutriments']['energy-kcal_100g']
+        protein = product['nutriments']['proteins_100g']
+        carbs = product['nutriments']['carbohydrates_100g']
+        sugars = product['nutriments']['sugars_100g']
+        fat = product['nutriments']['fat_100g']
+        saturated = product['nutriments']['saturated-fat_100g']
+    else:
+        continue
+
+    # these are optional
+    sodium = product['nutriments'].get('sodium_100g', None)
+    fibre = product['nutriments'].get('fiber_100g', None)
+    common_name = product.get('generic_name', None)
+    brand = product.get('brands', None)
+
+    source_name = "Open Food Facts"
+    source_url = f'https://world.openfoodfacts.org/api/v0/product/{code}.json'
+
+    lang_object = Language.objects.create(
+        short_name=lang,
+        full_name=langs[lang]
+    )
```

Comment:
Can you add the changes I mentioned on discord?

Reply:
Yep sorry, seems like an older version got uploaded for some reason!

Reply:
Changes got reverted somehow. Fixing now!